packages/babel-helper-create-class-features-plugin/src/features.js

```diff
@@ -13,7 +13,7 @@ const featuresSameLoose = new Map([
   [FEATURES.privateMethods, "@babel/plugin-proposal-private-methods"],
   [
     FEATURES.privateIn,
```

Comment:
```suggestion
  [FEATURES.privateIn, "@babel/plugin-proposal-private-property-in-object"],
```

There is a linting error